src/renderer/CharacterDisplayRenderer.cpp

```
@@ -92,11 +92,28 @@ void CharacterDisplayRenderer::draw(uint8_t byte) {
 
 void CharacterDisplayRenderer::drawBlinker() {
-    static_cast<CharacterDisplayInterface*>(display)->drawBlinker();
+    //    static_cast<CharacterDisplayInterface*>(display)->drawBlinker();
 }
 
 void CharacterDisplayRenderer::clearBlinker() {
-    static_cast<CharacterDisplayInterface*>(display)->clearBlinker();
+    //    static_cast<CharacterDisplayInterface*>(display)->clearBlinker();
+}
+
+bool CharacterDisplayRenderer::isBlinkerOn() {
```

Cppcheck (2.10-2)

[style] The function 'isBlinkerOn' is never used.

(unusedFunction)